Files/Views/Pages/PropertiesSecurity.xaml.cs

```diff
@@ -121,17 +116,5 @@ private async void OpenAdvancedProperties()
                 // Unsupported
             }
         }
-
-        private async void PropsView_Consolidated(ApplicationView sender, ApplicationViewConsolidatedEventArgs args)
-        {
-            Window.Current.Close();
-            propsView.Consolidated -= PropsView_Consolidated;
-            propsView = null;
-
-            if (SecurityProperties != null)
-            {
-                await Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () => SecurityProperties.GetFilePermissions()); // Reload permissions
```

Comment:
Nice work @winston-de, I'm only curious if you've discovered what of the Consolidated lambdas was causing the issue. If I were to add just the above code back do you already know if it will cause the issue?

Reply:
Sorry, I didn't realize this was here.
Anyway, yes, it's probably fine, the main issue seemed to be the Window.Current.Close() call. But it's probably worth testing in release mode just in case.